src/components/SplashScreen.js

```diff
@@ -1,62 +1,64 @@
 
 export default function SplashScreen(props) {
-    const [loading, setLoading] = useState(false)
-    const navigate = useNavigate()
+  const [loading, setLoading] = useState(false);
+  const navigate = useNavigate();
 
-    useEffect(() => {
-        setLoading(true)
-        setTimeout(() => {
-            setLoading(false)
-        }, 2000)
-    }, [])
+  useEffect(() => {
+    setLoading(true);
+    setTimeout(() => {
+      setLoading(false);
+    }, 2000);
+  }, []);
 
-    return (
-        <Box sx={{
-            display: 'flex',
-            flexDirection: 'column',
-            width: '100%',
-            height: '100vh',
-            background: 'linear-gradient(180deg, #e20e2d, #aa0f26)'
-        }}>
-            <div className='splashBG'>
-                <div className='logo'>
-                    <img className='image' src={logo} alt='Team Logo'/>
-                </div>
-                <div>
-                    {
-                        loading ?
-                            // While loading show loader image from react-spinners
-                            <div className='sweet-loading'>
-                                <ClipLoader color='#edc314' loading={loading} size={30} />
-                            </div>
-                        :
-                            //After loading is complete, show buttons
-                            <Slide direction='up' in={!loading}>
-                                <ButtonGroup orientation='vertical'>
-                                    <Button 
-                                        variant='contained'
-                                        color='primary'
-                                        sx={{m:2, width:300,height:50}}
-                                        onClick={() => {
-                                            navigate('/login')
-                                        }}
-                                    >
-                                        <Typography variant='h6'>
-                                            Sign In
-                                        </Typography>
-                                    </Button>
-                                </ButtonGroup>
-                            </Slide>
-                    }
-                </div>
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        width: "100%",
+        height: "100vh",
+        background: "linear-gradient(180deg, #e20e2d, #aa0f26)",
+        justifyContent: "center",
+        alignItems: "center",
+        textAlign: "center",
+      }}
+    >
+      <Box className="splashBG">
+        <div>
+          <img src={logo} alt="Team Logo" width={400} />
+        </div>
+        <div>
+          {loading ? (
+            // While loading show loader image from react-spinners
+            <div className="sweet-loading">
+              <ClipLoader color="#edc314" loading={loading} size={50} />
```

Comment:
Be careful with using hexcodes as props - what will you do if you need to change the color? Going through each place you use the color is gonna be a pain - configure your color palette using the MUI theme tooling